ruby/vampires.rb

```diff
@@ -0,0 +1,71 @@
+
+#Employee Total
+puts "How many employees"
+employee_total= gets.chomp.to_i
+#Loop 
+employee=0
+until employee == employee_total
+
+#Name
+puts "what's your name"
+name=gets.chomp.capitalize!
+
+#Age
+puts "how old are you"
+age =gets.chomp.to_i
+
+#Year
+puts "What year were you born"
+year= gets.chomp.to_i
+if 2016 - year == age
+  year= true
+else 
+  year=false
+end
+
+#Garlicbread question
+puts "Do you want garlic bread Y or N"
+garlic_bread = gets.chomp
+if garlic_bread =="y" 
+   garlic_bread=true
+elsif garlic_bread =="n"
+   garlic_bread=false
+else puts "put y or n"
+end
+
+#Insurance Question
+puts "Do you need health insurance, put Y or N"
+insurance = gets.chomp.upcase!
+if insurance =="Y"
+  insurance= true 
+elsif insurance == "N"
+  insurance=false
+else puts "put Y or N"
+end
+
+#Allergies Loop
+puts "do you have any allergies, list them and enter done when done"
+allergies=gets.chomp
+until allergies == "done"|| allergies =="sunshine"
+   puts "list your alergies, enter done when done"
+   allergies =gets.chomp
+end
+  
+if name == "Drakula" || name == "TuFang"
+  puts "definently a vampire"
+elsif allergies == "sunshine"
+  puts "probaly a vampire"
+elsif year == true  && (garlic_bread == true||insurance == true)
+  puts "probaly not a vampire"
+elsif year == false && (garlic_bread == false && insurance == false) 
+  puts "almost certanly a vampire"
```

Comment:
In the comment about refactoring to go in the order the facts were presented in, I also mentioned setting the result of the if statements to a variable like `result` equal to a string representing the outcome of the survey. Try refactoring with that, since `puts` can also be a tricky method. 
`puts` is an interesting method, it takes an object as an argument, displays it to `$STDOUT` (your terminal), and returns `nil`. The thing to remember with puts is that it returns `nil` after it is called, so when we write code like
```
def add
	puts 1 + 2
end
```
we think we are returning 3 when we call the method because it displays to our screen, but we actually get `nil` (as the the last evaluated expression is the return of a method).
To avoid this, just omit `puts` from the method. Now you may say, 'But how do we know it's working?' Great question! For starters we could call our method and display its return with the more friendly development-only method `p` like so.
	`p add`
The gist is, only use `puts` when you want to display text to the screen, if your methods are supposed to do something else, let them and if you are curious look at the results outside of that method.